core-libs/setup/ssr/optimized-engine/ssr-optimization-options.ts

```diff
@@ -110,3 +110,10 @@ export enum RenderingStrategy {
   DEFAULT = 0,
   ALWAYS_SSR = 1,
 }
+
+export const defaultSsrOptimizationOptions: SsrOptimizationOptions = {
+  concurrency: 20,
+  timeout: 3_000,
+  maxRenderTime: 300_000,
```

Comment:
maxRenderTime - is it a new default added by the way?

Reply:
No, it was set to 300000 ms (5 mins) from the start.
We are just making it more obvious by explicitly setting it here.